### PR TITLE
<onion-video>

### PR DESCRIPTION
**Pull request by collin**

### CODE REVIEW

File: elements/bulbs-video/bulbs-video-cms.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  registerElement,
+  EmbededCMSElement,
+} from 'bulbs-elements/register';
+
+class EmbeddedBulbsVideo extends EmbededCMSElement {
+  get embedContentPreview () {
+    return `
+      <h1 style='text-align: center; font-family: "Droid Serif"'>
+        <i class='fa fa-puzzle-piece'></i>
+        Embedded BulbsVideo
```
Comment (spra85) [inline elements/bulbs-video/bulbs-video-cms.js]: Wonder if we should nip this early so we don't have the same problem as the polls element - maybe we have this embedded CMS element make a request to the Video Hub to get the video title and display it?

Comment (spra85) [inline elements/bulbs-video/bulbs-video-cms.js]: Realize this is essentially a stub element...but thinking ahead

Comment (collin) [inline elements/bulbs-video/bulbs-video-cms.js]: Yeah, that makes sense. It should be a very simple data fetch.

**Comment by jmelvnsn** [comment]
@collin https://github.com/theonion/bulbs-elements/pull/1/commits/06bebe25342397ae1b8bb95ba60d3b2a5b2a4e79 Not perfect / but a direction

**Comment by collin** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;9KdFM855&#x2F;90-flash-enabled-false-type-things-belongs-in-bulbs-elements"> flash enabled false type things - belongs in bulbs elements</a>

**Comment by collin** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;NPvjzlI5&#x2F;1-base-functionality-for-the-main-player-video-meta-component"> base functionality for the main player (video meta component)</a>

**Comment by jmelvnsn** [comment]
@collin @spra85 Have a looksy. Need to get this out in advance of AVC video detail page

**Comment by collin** [comment]
@jmelvnsn We'll need to re-run travis once videohub-player is released.

**Comment by jmelvnsn** [comment]
@spra85 @collin How are we looking here?

**Comment by jmelvnsn** [comment]
@kand Please to bless when you have a minute

**Comment by kand** [comment]
Oh man this is gonna take more than a minute. Can I look at this tomorrow?

File: elements/bulbs-video/bulbs-video.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import 'videohub-player/dist/videohub-player.css';
+import 'videojs-autoplay-toggle/videojs.autoplay-toggle.css';
+
+export default class BulbsVideo extends BulbsElement {
+  initialDispatch () {
+    this.store.actions.fetchVideo(this.props.src);
+  }
+
+  componentWillReceiveProps (props) {
+    // Prop Will Change
```
Comment (daytonn) [inline elements/bulbs-video/bulbs-video.js]: This comment should probably be removed. I'm not even sure what it's saying.

**Comment by daytonn** [comment]
Looks good 👍

**Pull request merged**